alpha/nodejs-chart/README.md

```diff
@@ -0,0 +1,66 @@
+# Node.js Helm Chart
+A Helm chart for building and deploying a [Node.js](https://nodejs.org/) application on OpenShift.
+
+## Prerequisites
+You will need to create a pull secret if you pull an S2I builder or Docker base image from an external registry. Use the following command as a reference to create your pull secret:
+```bash
+oc create secret docker-registry my-pull-secret --docker-server=$SERVER_URL --docker-username=$USERNAME --docker-password=$PASSWORD --docker-email=$EMAIL
+```
+
+You can use this secret by passing `--set build.pullSecret=my-pull-secret` to `helm install`, or you can configure this in a values file:
+```yaml
+build:
+  pullSecret: my-pull-secret
+```
+and apply by passing `--values $VALUES_FILE`.
+
+## Values
+This section describes the Values used to configure this chart.
+
+### Required Values 
+Below shows the required values to install this chart.
+
+| Value | Description | Default | Additional Information |
+| ----- | ----------- | ------- | ---------------------- |
+| `build.uri` | Git URI that references your git repo | - | **REQUIRED** |
```

Comment:
This isn't yet merged in for the Quarkus chart, but #40 sets a default build.uri for Quarkus so users can just install right off the shelf to build a quickstart application. We're going to want to do this here as well.